### PR TITLE
Move AdditionalRegistrationFields to models.py

### PR DESCRIPTION
**Pull request by danialmalik**
#### Story Link
https://edlyio.atlassian.net/browse/EDE-329

#### PR Description

While running the tests Open edX creates database based on models
in models.py files of all of the apps. Putting this model in models.py
fixes the failing tests.

##### Before (42 failures)
image

##### After (40 failures)
image

#### Type of change

Please select the options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Change

#### Checklist before merging:

- [x] Squased
- [ ] Reviewd

### CODE REVIEW

**Comment by danialmalik** [comment]
jenkins run python

**Comment by edtecheco** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

File: openedx/features/ucsd_features/models.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# Backwards compatible settings.AUTH_USER_MODEL
+USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
+
+
+class AdditionalRegistrationFields(models.Model):
+    """
+    This model contains two extra fields that will be saved when a user registers.
+    The form that wraps this model is in the forms.py file.
+    """
+    user = models.OneToOneField(USER_MODEL, null=True)
```
Comment (HamzaIbnFarooq) [inline openedx/features/ucsd_features/models.py]: Are we sure that we need null to be true here? Secondly, If we have to allow null than I think that should only be the scenario where on_delete=models.SET_NULL.

File: openedx/features/ucsd_features/models.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    gender_nb = models.CharField(blank=False, max_length=32, verbose_name=b'Gender', choices=[
+        (b'male', b'Male'),
```
Comment (HamzaIbnFarooq) [inline openedx/features/ucsd_features/models.py]: Is there any specific reason for using bytes here?

File: openedx/features/ucsd_features/models.py (hunk 1 above)
Comment (danialmalik) [reply openedx/features/ucsd_features/models.py]: Team UCSD code :D

File: openedx/features/ucsd_features/models.py (hunk 2 above)
Comment (danialmalik) [reply openedx/features/ucsd_features/models.py]: No idea.

File: openedx/features/ucsd_features/models.py (hunk 1 above)
Comment (HamzaIbnFarooq) [reply openedx/features/ucsd_features/models.py]: ok, but keep in mind that git blame doesn't know that.

Comment (danialmalik) [reply openedx/features/ucsd_features/models.py]: I've added `on_delete=models.SET_NULL` constraint though.

**Pull request merged**